discovery/roles/discovery_validations/common/tasks/include_high_availability_config.yml

```diff
@@ -28,6 +28,15 @@
   ansible.builtin.set_fact:
     enable_oim_ha: "{{ high_availability_config.ansible_facts.oim_ha.enable_oim_ha | default(false) }}"
 
+- name: Set OIM admin IP to virtual IP if OIM HA is enabled
+  when: enable_oim_ha
+  ansible.builtin.set_fact:
+    network_data: "{{ network_data | combine(update_data, recursive=true) }}"
```

Comment:
instead of overriding the input value override admin ip and update the place using the fqdn path with admin_ip